superset/assets/src/SqlLab/App.jsx

```diff
@@ -32,10 +31,16 @@ const store = createStore(
   ),
 );
 
-// jquery hack to highlight the navbar menu
-$('a:contains("SQL Lab")')
-  .parent()
-  .addClass('active');
+// Highlight the navbar menu
+const menus = document.querySelectorAll('.nav.navbar-nav li.dropdown');
+const sqlLabMenu = Array.prototype.slice.apply(menus)
+  .filter(element => element.innerText.trim() === 'SQL Lab')[0];
```

Comment:
[].filter(cb)[0] is equivalent to [].find(cb)?
also if your filter function returns [], get its [0] will make trouble.

Reply:
Thank you. Will update.